<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QWidget" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QLineEdit" name="lEdt_ip">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.1.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdt_port">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1234</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn_Connect">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Conectar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn_Close">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Message">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mensaje recibido</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
